types/office-js/index.d.ts

```diff
@@ -14782,9 +14790,7 @@ declare namespace Office {
         /**
          * Adds an event handler for a supported event.
          *
-         * Currently the only supported event type is Office.EventType.ItemChanged, which is invoked when the user selects a new item. 
-         * This event is used by add-ins that implement a pinnable taskpane, and allows the add-in to refresh the taskpane UI based on the currently 
-         * selected item.
+         * Currently the supported event types are `Office.EventType.ItemChanged` and `Office.EventType.OfficeThemeChanged`.
```

Comment:
Either "Currently, the supported event types..." or "The currently supported event types..."